core/src/services/routing.js

```diff
@@ -299,3 +298,24 @@ const showPageNotFoundError = async (
   component.set({ alert });
   navigateTo(GenericHelpers.addLeadingSlash(pathToRedirect));
 };
+
+export const navigateToLink = item => {
```

Comment:
I think the name `item` is way too generic.  Should it be `node` maybe?

Reply:
I think item is good enough, this is not a Luigi node necessarily, it could be an item in the profile dropdown or an item in the product switcher.